design_docs/template.md

```diff
@@ -0,0 +1,67 @@
+<h1> Title: Some descriptive title <br>
+MPAS-Analysis Team <br>
+date: x - xx - xxxx <br>
+</h1>
+<h2> Summary </h2>
+The purpose of this section is to summarize what capability is to be added to
+the ACME MPAS-Analysis system through this design process. It should be clear
```

Comment:
I would suggest deleting ACME here since MPAS-Analysis should be able to live outside of ACME